devices.js

```diff
@@ -2906,9 +2906,9 @@ const devices = [
         model: '1TST-EU',
         vendor: 'eCozy',
         description: 'Smart heating thermostat',
-        supports: 'temperature, occupancy, un-/occupied heating, schedule',
+        supports: 'temperature, battery, occupancy, un-/occupied heating, schedule',
         fromZigbee: [
-            fz.legacy_battery_voltage,
+            fz.wiser_itrv_battery,
```

Comment:
Can you try using `fz.battery` here, if that doesn't work you may need to add the following to `configure`: `await configureReporting.batteryPercentageRemaining(endpoint);`

Reply:
It seems that the device give a voltage in the range between 2.2V and 3V. I thought using a similar implementation as the one used by the Wiser ITRV unit.

Reply:
Yes, but if we can make the device to directly report the percentage would be more accurate.

Reply:
I don't see this reported in the debug info. How can I find out if the device is able to report this?

Reply:
I forgot to mention that after this change (https://github.com/Koenkk/zigbee-herdsman-converters/pull/1310#discussion_r439729952) you have to force a reconfigure via https://www.zigbee2mqtt.io/information/mqtt_topics_and_message_structure.html#zigbee2mqttbridgeconfigure). If the zigbee2mqtt shows `Configured successfully` the device likely supports it.

Reply:
I'm a bit puzzled by this eCozy device documentation. Most of the commands that are described in the documentation don't seem to work:
```
zigbee2mqtt:debug 2020-06-14 16:18:14: Received MQTT message on 'zigbee2mqtt/0x70b3d5de0000106c/get' with data '{
  "pi_heating_demand": ""
}'
zigbee2mqtt:error 2020-06-14 16:18:14: No converter available for 'pi_heating_demand' ()
```

Reply:
Could it be that the documentation is based on changes that were never pulled into your repo? When I look at the repo off the dev that worked on it I find in devices.js:
```
 {
        zigbeeModel: ['Thermostat'],
        model: '1TST-EU',
        vendor: 'eCozy',
        description: 'Smart heating thermostat',
        supports: 'temperature, occupancy, un-/occupied heating, schedule',
        fromZigbee: [
            fz.ignore_basic_change, fz.generic_battery_voltage,
            fz.thermostat_att_report, fz.thermostat_dev_change,
        ],
        toZigbee: [
            tz.factory_reset, tz.thermostat_local_temperature, tz.thermostat_local_temperature_calibration,
            tz.thermostat_occupancy, tz.thermostat_occupied_heating_setpoint,
            tz.thermostat_unoccupied_heating_setpoint, tz.thermostat_setpoint_raise_lower,
            tz.thermostat_remote_sensing, tz.thermostat_control_sequence_of_operation, tz.thermostat_system_mode,
            tz.thermostat_weekly_schedule, tz.thermostat_clear_weekly_schedule, tz.thermostat_weekly_schedule_rsp,
            tz.thermostat_relay_status_log, tz.thermostat_relay_status_log_rsp,
        ],
        configure: (ieeeAddr, shepherd, coordinator, callback) => {
            const device = shepherd.find(ieeeAddr, 3);
            const actions = [
                // from https://github.com/ckpt-martin/Hubitat/blob/master/eCozy/eCozy-ZigBee-Thermostat-Driver.groovy
                (cb) => device.bind('genBasic', coordinator, cb),
                (cb) => device.bind('genPowerCfg', coordinator, cb),
                (cb) => device.bind('genIdentify', coordinator, cb),
                (cb) => device.bind('genTime', coordinator, cb),
                (cb) => device.bind('genPollCtrl', coordinator, cb),
                (cb) => device.bind('hvacThermostat', coordinator, cb),
                (cb) => device.bind('hvacUserInterfaceCfg', coordinator, cb),
                (cb) => device.report('hvacThermostat', 'localTemp', 5, 30, 0, cb),
            ];

            execute(device, actions, callback);
        },
    },
```
Source: https://github.com/sebastianheierhoff/zigbee-shepherd-converters/blob/dev/devices.js

Reply:
Not sure, but if that works OK I'm happy to merge that